feconf.py

```diff
@@ -453,6 +453,12 @@ def get_empty_ratings():
 # Mailchimp username.
 MAILCHIMP_USERNAME = None
 
+# If requests can be sent to the bulk email provider (requires the Mailchimp API
+# key, username and audience id fields to be filled).
+CAN_SEND_REQUESTS_TO_BULK_EMAIL_SERVICE_PROVIDER = bool(
+    (not constants.DEV_MODE) and MAILCHIMP_AUDIENCE_ID and MAILCHIMP_API_KEY and
```

Comment:
@vojtechjelinek just to check, is `not constants.DEV_MODE` correct here? I know we have dev mode, emulator mode and prod mode; we only want this to happen in prod. What should be the correct invocation?

Reply:
probably better to use `not constants.DEV_MODE and not constants.EMULATOR_MODE`

Reply:
Removed this new flag as per earlier comment.